xtask/src/main.rs

```diff
@@ -136,8 +137,13 @@ fn test_cross(deny_warnings: bool) {
         false => vec![],
     };
 
+    let mut features = vec!["", "alloc"];
```

Comment:
😬 I don't love the empty string but I see why it exists below, so it's fine. It just... stinks.

Reply:
It does. In the past I tested `vec![None, Some("alloc")]`, which feels more clean but adds quite a bit of boilerplate.